syntax = "proto3";
import "ml2/grpc/trace/trace.proto";

message LTLEquivProblem {
  string formula1 = 1;
  string formula2 = 2;
  optional float timeout = 3;
}

message LTLEquivSolution {
  string status = 1;
  optional float time = 2;
  optional Trace exclusive_word = 3;
}